// expected error: error: function call received the wrong number of parameters (expected 1, received 2)
// unexpected output: FAIL

println('FAIL');

Null foo(Integer value) { }

foo(1, 2);